tests_smoke/smoke/test_api_one_off_file_link.py

```diff
@@ -0,0 +1,34 @@
+import requests
+
+from .common import Config, pretty_print, single_succeeded
+
+
+def test_api_one_off_file_link():
+
+    print("test_api_one_off_file_link (email)... ", end="", flush=True)
+
+    data = {
+        "email_address": Config.EMAIL_TO,
+        "template_id": Config.EMAIL_TEMPLATE_ID,
+        "personalisation": {
+            "var": {"file": "aGkgdGhlcmU=", "filename": "test_file.txt", "sending_method": "link"},
```

Comment:
It's almost the same code except for the `sending_method`, no? (and console messages)

Reply:
yeah, good point. I like to keep tests for different things separate, but in this case we're really testing three variants of one api call. 👍